Configurations/windows-makefile.tmpl

```diff
@@ -160,6 +160,7 @@ ENGINESDIR=$(ENGINESDIR_dev)$(ENGINESDIR_dir)
 !ENDIF
 
 CC={- $config{cc} -}
+CPP={- $config{cpp} -}
```

Comment:
On side note one *probably* has to recognize that nmake pre-defines CPP but is has different meaning, CPlusPlus, not CPre-Processor. So that formally speaking a hard-core nmake user might view this as confusing/inappropriate. Fortunately there probably aren't any around and we can get away without somebody rising eyebrows. As it does work since we don't rely on implicit rules...

Reply:
Oh!  I was wondering why they had such a wonky definition...  that explains it.  We *could* adapt, it wouldn't even be hard, but we then need to decide on a name to replace CPP

Reply:
Well, one can as well argue that nmake should blame itself. As problem runs even deeper. Implicit pre-defined nmake rule that uses $(CPP) as reference to CPlusPlus compiler even uses $(CPPFLAGS), also as CPlusPlus flags. So that if one aims for making it less confusing for nmake user, one is going to end up making it more confusing to a lot of Unix make users. And [as already said] there are no hard-core nmake users, so I'd say it's more appropriate to make it slide as it is... Let's just try to keep this in mind as possible caveat...

Reply:
Works for me